apcd-cms/src/client/src/components/Forms/Registrations/TextFormField.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Field, ErrorMessage } from "formik";
+import {
+    FormGroup,
+    Label,
+    FormFeedback,
+  } from "reactstrap";
+  
+/* Component to re-use for text input fields */
+export const TextFormField: React.FC<{name: string; label: any; helpText?: any; required?: boolean}> = ({name, label, helpText, required}) => {
+    return (
+        <FormGroup 
+            className={`field-wrapper textinput ${ required ? 'required': '' }`}
+            noMargin={true}
+        >
+            <Label htmlFor={name}>
+                {label}
+                {required ? <span style={{"color": "red"}}> (required)</span> : <></>}
+            </Label>
+            <Field name={name} as="input" id={name} className="textinput"/>
+            <ErrorMessage
+                name={name}
+                component={FormFeedback}
+            />
```

Comment:
```suggestion
                    component="div"
                    className={styles.isInvalid}
                  />
```

If you change all error messages to the above code, and add a style like this to your css module


```
.isInvalid {
  width: 100%;
  margin-top: 0.25rem;
  font-size: 80%;
  color: #dc3545;
}
```

You should now see the validation errors with similar styles to FormFeedback from react strap